src/oca_pre_commit_hooks/global_parser.py

```diff
@@ -57,6 +57,12 @@ def __init__(self):
         self.add_argument(
             "--list-msgs", default=False, action="store_true", help="List all currently enabled messages."
         )
+        self.add_argument(
+            "--autofix",
```

Comment:
Almost of the hooks are using only `--fix`
e.g.
 - https://github.com/pre-commit/mirrors-eslint
 - https://github.com/pre-commit/pre-commit-hooks


Could we use the same way?
could we